sdk/go/auto/errors.go

```diff
@@ -129,6 +133,9 @@ func IsRuntimeError(e error) bool {
 	if strings.Contains(as.Error(), "go inline source runtime error") {
 		return true
 	}
+	if as.code != 0 {
+		return true
+	}
```

Comment:
This method has specific strings it searches for and to make it more resilient, we check the status code.